sdk/data/azcosmos/emulator_cosmos_item_test.go

```diff
@@ -503,6 +503,211 @@ func TestItemIdEncodingComputeGW(t *testing.T) {
 	verifyEncodingScenario(t, container, "ComputeGW-IdWithUnicodeCharacters", "WithUnicode鱀", http.StatusCreated, http.StatusOK, http.StatusOK, http.StatusNoContent)
 }
 
+func TestItemCRUDHierarchicalPartitionKey(t *testing.T) {
```

Comment:
I dug through the [test logs](https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/4238297/logs/180) and this test is failing under the emulator.
```
2024-10-16T14:21:15.4860400Z RESPONSE 400: 400 Bad Request
2024-10-16T14:21:15.4861044Z ERROR CODE: 400 Bad Request
2024-10-16T14:21:15.4862521Z --------------------------------------------------------------------------------
2024-10-16T14:21:15.4863133Z {
2024-10-16T14:21:15.4864052Z   "code": "BadRequest",
2024-10-16T14:21:15.4865785Z   "message": "Message: {\"Errors\":[\"Too many partition key paths (2) specified. A maximum of 1 is allowed.\"]}\r\nActivityId: 564a7bbc-66a2-4ad1-8da6-be65731282cb, Request URI: /apps/DocDbApp/services/DocDbMaster0/partitions/780e44f4-38c8-11e6-8106-8cdcd42c33be/replicas/1p/, RequestStats: , SDK: Microsoft.Azure.Documents.Common/2.14.0"
2024-10-16T14:21:15.4866735Z }
```